### PR TITLE
Update README.MD to include 1 liner Mac installation

### LINKED ISSUE #23: Readme installation instruction only applicable on Linux
Following the instructions here https://github.com/pantheon-systems/sites-yml-validator#latest-version is not working for mac

### PR DESCRIPTION
**Pull request by carl-alberto**
closes #23

### CODE REVIEW

**Review by jazzsequence — DISMISSED** [review]
lgtm but would like @pwtyler's thoughts as well

File: README.MD [hunk 1]
```diff
@@ -36,6 +36,7 @@ A `sites.yml` file is valid if:
 
 Use wget to download, gzipped pre-compiled binaries:
 For instance, VERSION=v0.0.11 and BINARY=sites-yml-validator_linux_amd64
+Refer to the [Releases](https://github.com/pantheon-systems/sites-yml-validator/releases) page for `VERSION` and `BINARY` parameteres
```
Comment (pwtyler) [inline README.MD:39]:
```suggestion
Refer to the [Releases](https://github.com/pantheon-systems/sites-yml-validator/releases) page for `VERSION` and `BINARY` parameters
```

**Review by pwtyler — APPROVED** [review]

**Comment by pwtyler** [comment]
Ah, I forgot tests won't run on forks, I was sitting here waiting for tests to pass...

**Comment by pwtyler** [comment]
Pulled in on #26.

**Pull request closed without merging**